metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/AliasService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.usermetadata;
+
+import com.netflix.metacat.common.QualifiedName;
+import lombok.NonNull;
+
+/**
+ * Alias Service API.
+ *
+ * @author rveeramacheneni
+ * @since 1.3.0
+ */
+public interface AliasService {
+    /**
+     * Returns the original table name if present.
+     *
```

Comment:
I thought the intention is to return the alias name ( if it has one) instead of the original name.

Reply:
Ah no, it's the opposite (I guess it depends on your frame of reference :)). A request for `hive/default/table` should be mapped to an original name stored in Metacat like `hive/default/table_iceberg` .